development/github-desktop/github-desktop.SlackBuild

```diff
@@ -27,7 +27,7 @@ cd $(dirname $0) ; CWD=$(pwd)
 PRGNAM=github-desktop
 SRCNAM=GitHubDesktop
 VERSION=${VERSION:-3.3.12}
-BUILD=${BUILD:-1}
+BUILD=${BUILD:-2}
```

Comment:
The build suffix is not upstream build numbers, they are for changes to the slackbuild. Please use another flag for tracking upstream versions.